hotswap-agent-core/src/main/java/org/hotswap/agent/config/PluginConfiguration.java

```diff
@@ -182,7 +188,7 @@ private void loadConfigurationFile() {
     protected void init() {
         LogConfigurationHelper.configureLog(properties);
         initPluginPackage();
-
+        initIncludedClassLoaderPatterns();
```

Comment:
'include' and 'exclude' should be exclusive to each other, otherwise it doesn't make sense. We should check and throw exception to stop running.